sensing/autoware_pointcloud_preprocessor/config/filter.param.yaml

```diff
@@ -0,0 +1,8 @@
+/**:
+  ros__parameters:
```

Comment:
Extra file?

Reply:
Thanks! added it because the launch file needed it but since `filter` still has default values so removing it is fine.
removed in https://github.com/autowarefoundation/autoware.universe/pull/8468/commits/f511fc2b24df4e8a77384b06ec8c387a5fcbc6c0